geps/gep-1748.md

```diff
@@ -0,0 +1,140 @@
+creating multiple Multi-Cluster Services, one for each subset of endpoints you
+would like to route to. For example, the following configuration will send
+requests with paths prefixed with “/west” to the store-west ServiceImport, and
+“/east” to the store-east ServiceImport. Requests that don’t match either of
+these paths will be routed to the “store” ServiceImport which represents a
+superset of the “store-west” and “store-east” ServiceImports.
+
+```yaml
+{% include 'standard/multicluster/httproute-location.yaml' %}
+```
+
+#### Advanced Routing With ServiceImports
+
+All Routing capabilities in Gateway API should apply equally whether the backend
+is a Service or ServiceImport. For example, when routing to a system with
+multiple read replicas, it could be beneficial to route requests based on HTTP
+Method. In the following example, requests with POST, PUT, and DELETE methods
+are routed to `api-primary` while the rest are routed to `api-replicas`:
+
+```yaml
+{% include 'standard/multicluster/httproute-method.yaml' %}
+```
+
+#### Routing to Both Services and ServiceImports
+
+There are some situations where it will be useful to split traffic between a
+Service and ServiceImport. In the following example, 90% of traffic would go to
+endpoints attached to the cluster-local "store" Service, and the remaining 10%
+would go to endpoints attached to the Multi-Cluster "store-global" Service:
+
+```yaml
+{% include 'standard/multicluster/httproute-hybrid.yaml' %}
+```
+
+
+#### Cross-Namespace References with ReferenceGrant
+
+It is possible to use ReferenceGrant to enable cross-namespace references to
+ServiceImports. For example, the following HTTPRoute would forward traffic to
+endpoints attached to the “bar” Service in the “bar” namespace:
+
+```yaml
+{% include 'standard/multicluster/httproute-referencegrant.yaml' %}
+```
+
+### Mesh: ServiceImport as Parent
+
+In some cases, you may want to override traffic destined for a Multi-Cluster
+Service with a mesh. As part of the broader GAMMA initiative, ServiceImport can
+be used in the same way that Service is used as a ParentRef. When a Service is
+specified as a parent, meshes will intercept traffic destined for the ClusterIP
+of the Service and apply any policies or routing decisions defined by the Route.
+Similarly, when a ServiceImport is specified as a parent, meshes will intercept
+traffic destined for the ClusterSetIP and apply any policies or routing
+decisions defined by the Route. In the following example, the mesh would
+intercept traffic destined for the store ClusterSetIP and redirect it to
+endpoints for the local Service within the same cluster.
+
+```yaml
+{% include 'standard/multicluster/httproute-gamma.yaml' %}
+```
+
+### Services vs ServiceImports
+
+It is important that all implementations provide a consistent experience. That
+means that references to Services MUST always be interpreted as references to
```

Comment:
I share John's concerns above; I'd be much more comfortable with the MUST being lessened to a SHOULD at least until MCS API is beta.

Reply:
Yeah I think that's a fair point. Hopefully MCS API is going to make it to beta sometime soon, but agree that we should not overly restrict Services until that point. Updated to "SHOULD".